src/metatrain/experimental/nanopet/modules/attention.py

```diff
@@ -0,0 +1,69 @@
+import torch
+
+
+class AttentionBlock(torch.nn.Module):
```

Comment:
could this say why we are not using the standard attention in torch? I remember there was a reason, but I don't remember the reason